DFC.Api.Content.UnitTests/DFC.Api.Content.UnitTests.csproj

```diff
@@ -1,13 +1,33 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <IsPackable>false</IsPackable>
     <ProjectGuid>{3998DD4A-3D9B-407C-B4B4-16DF16515ED5}</ProjectGuid>
   </PropertyGroup>
   <ItemGroup>
+    <None Remove="Files\JobProfileRecordResponse_1.json" />
```

Comment:
It'd be good to copy using * here perhaps, as I'm always forgetting to set the properties on files 😄. i.e.
```
      <Content Include="Files\*">
        <CopyToOutputDirectory>Always</CopyToOutputDirectory>
      </Content>

```